lib/rspec/core/configuration_options.rb

```diff
@@ -169,7 +169,8 @@ def parse_args_ignoring_files_or_dirs_to_run(args, source)
       def args_from_options_file(path)
         return [] unless path && File.exist?(path)
         config_string = options_file_as_erb_string(path)
-        FlatMap.flat_map(config_string.split(/\n+/), &:shellsplit)
+        config_lines = config_string.split(/\n+/).reject { |s| s.match?(/\A\s*#/) }
```

Comment:
`match?/1` isn't supported by all of our supported Rubies, this would need to change to be merged,

Reply:
Thank you for your review. I use `=~` instead: https://github.com/rspec/rspec-core/pull/2984/commits/51d6054c3a0639786ea2bc9f844bf62c80d7338d